src/leodb/db.cpp

```diff
@@ -12,6 +14,7 @@ bool DB<T, U>::put(Key<T> key, Value<U> value) {
     try {
```

Comment:
try catch logic doesn't make sense since it doesnt error out

Reply:
Removed